tests/utils/test_process.py

```diff
@@ -0,0 +1,116 @@
+# SPDX-FileCopyrightText: 2024-present Datadog, Inc. <dev@datadoghq.com>
+#
+# SPDX-License-Identifier: MIT
+from __future__ import annotations
+
+import os
+import sys
+
+from deva.utils.process import EnvVars
+
+
+def get_random_name():
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Would it be possible to make this deterministic? I'm not sure that tests are run in order so maybe we can hash an argument instead of providing a random value. I think it is proper to have deterministic tests to debug

Reply:
The tests are randomized https://github.com/DataDog/datadog-agent-dev/pull/2/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R48

The goal here is to use a random environment variable because the utility modifies the entire process.